README.md

```diff
@@ -132,6 +132,12 @@ doctrine:
         url: '%env(ekino_encrypted:DATABASE_URL)%'
 ```
 
+## Encrypt passwords
+
+To encrypt your passwords, run the following command:
```

Comment:
```suggestion
To encrypt your secret, run the following command:
```

Reply:
I think this will cause confusion between the secret (text to encrypt) and the secret (key to encrypt the text).

Reply:
See new version using "text".